packages/react-sdk/src/remix.ts

```diff
@@ -0,0 +1,47 @@
+const getRemixSegment = (segment: string) => {
+  if (segment === "*") {
+    return "$";
+  }
+  const match = segment.match(/^:(?<name>\w+)(?<modifier>\*|\?)?$/);
```

Comment:
I would give a comment example what we are matching here
And probably that we don't support various original patterns
i.e.
`https://example.com/jo*` in our case should throw instead of `$` isn't it?

Reply:
We validate this case out in builder